board/safety/safety_subaru.h

```diff
@@ -118,9 +134,11 @@ addr_checks subaru_gen2_rx_checks = {subaru_gen2_addr_checks, SUBARU_GEN2_ADDR_C
 
 const uint16_t SUBARU_PARAM_GEN2 = 1;
 const uint16_t SUBARU_PARAM_LONGITUDINAL = 2;
+const uint16_t SUBARU_PARAM_LKAS_ANGLE = 4;
 
 bool subaru_gen2 = false;
 bool subaru_longitudinal = false;
+bool lkas_angle = false;
```

Comment:
any branding from Subaru denoting this change? For Toyota the move was from LKAS to LTA for example.

Reply:
haven't seen any specific branding changes. I renamed it to subaru_lkas_angle though